GTG/gtk/browser/main_window.py

```diff
@@ -1285,9 +1285,12 @@ def on_dismiss_task(self, widget=None):
     def apply_filter_on_panes(self, filter_name, refresh=True, parameters=None):
         """ Apply filters for every pane: active tasks, closed tasks """
```

Comment:
Isn't this docstring a lie now? (and the one in unapply_filter_on_panes too)

Reply:
Well yeah, I'll adjust the docstring, but indeed, by default with the `force_refresh=False` all pane _won't_ be applied filters upon.